.travis.yml

```diff
@@ -41,8 +43,6 @@ matrix:
     env: MONGODB=${MONGODB_3_4} PYMONGO=3.4.x
   - python: 3.6
```

Comment:
Perhaps we can bump the Python version here to 3.7? I think of this configuration as "test the latest supported Python version, the latest supported MongoDB version, and the latest supported PyMongo version".